libparsec/crates/platform_mountpoint/src/unix/mount.rs

```diff
@@ -150,10 +153,8 @@ impl Drop for Mountpoint {
 // changes on the mountpoint path.
 fn create_suitable_mountpoint_dir(
     base_mountpoint_path: &std::path::Path,
-    ops: &WorkspaceOps,
+    name: &EntryName,
```

Comment:
I'd keep `workspace_name` to better communicate the intent
```suggestion
    workspace_name: &EntryName,
```